plugins/modules/rdfgroup.py

```diff
@@ -441,6 +441,7 @@ def get_rdf_volume_list(self, rdf_number):
             msg = ('Get RDF Volumes List for RDF Group %s failed with error %s' %
                    (rdf_number, str(e)))
             self.show_error_exit(msg=msg)
+            return []
```

Comment:
`show_error_exit` will call `fail_json`, that will terminate the process of ansible.
I think no need to `return []`.
BTW, the [fail_json](https://github.com/dell/ansible-powermax/blob/main/tests/unit/plugins/module_utils/shared_library/fail_json.py#L20) used by UT cannot mock the real behavior of ansible's fail_json. 
Please refer to the PowerScale [PR ](https://github.com/dell/ansible-powerscale/pull/173/) for the correct mock.
In short, the key point is
`self.powerscale_module_mock.module.fail_json = MagicMock(
            side_effect=SystemExit)`